ksp_plugin_adapter/ksp_plugin_adapter.cs

```diff
@@ -844,11 +880,20 @@ where is_in_physics_bubble(vessel)
       rendered_flight_plan_[i] = NewRenderedTrajectory(
           (i % 2 == 0) ? XKCDColors.RoyalBlue : XKCDColors.OrangeRed);
     }
+    rendered_frenet_trihedra_ = new VectorLine[3 * (segments / 2)];
+    for (int i = 0; i < segments / 2; ++i) {
+      rendered_frenet_trihedra_[3 * i] =
+          NewUILine(new UnityEngine.Color(0.84f, 1, 0));
```

Comment:
Done, it was hard to find an utility to convert RGB values to the nearest xkcd colour though.